spec/s3_meta_sync_spec.rb

```diff
@@ -16,7 +17,8 @@ def upload_simple_structure
   end
 
   def download(file)
-    open("https://s3-us-west-2.amazonaws.com/#{config[:bucket]}/#{file}").read
+    url=URI.parse("https://s3.amazonaws.com/#{config[:bucket]}/#{file}")
```

Comment:
no it doesn't work without URI parser. It would return smth like "file not found"